solution/ui/regulations/eregs-vite/src/views/Search.vue

```diff
@@ -355,24 +355,22 @@ export default {
         },
         async retrieveResourcesResults({ query, page, pageSize }) {
             try {
-                const isSearchGov = window.location.href.match("searchgov");
-
                 const commonParams = {
                     q: query,
                     page,
                 };
 
-                const djangoParams = {
-                    ...commonParams,
-                    partDict: "all",
-                    page_size: pageSize,
-                    fr_grouping: false,
-                };
-
-                const response = isSearchGov
-                    ? await getSearchGovResources(commonParams)
-                    : await getSupplementalContent(djangoParams);
+                let response = await getSearchGovResources(commonParams)
 
+                if(response.count === 0){
```

Comment:
These new lines need to be linted.  (Needs trailing semicolon, space between `if` and the condition, etc)

Reply:
linted